cypress/integration/pages/onDemandRadio/testsForCanonicalOnly.js

```diff
@@ -12,5 +14,45 @@ export default ({ service, pageType }) => {
         });
       }
     });
+    describe('Radio Schedule', () => {
+      it('should be displayed if there is enough schedule data', () => {
+        const isRadioScheduleOnPage = serviceHasRadioSchedule({
```

Comment:
As discussed - atm this `serviceHasRadioSchedule` helper is hard coded to look for the `onLiveRadioPage` toggle. It could be adapted to take in a third argument, or a separate helper created for OD radio.

Reply:
Adapted

Reply:
👍